test/functional/fixtures/regression/t171129/test.js

```diff
@@ -0,0 +1,3 @@
+it('T171129 - MVCxImageAndDataNavigationDemos/ImageSlider/VideoGallery demo page', function () {
```

Comment:
You can call it something like `test hangs on the MVCxImageSlider demo page (T171129)`